resources/assets/js/quick-links.js

```diff
@@ -20,3 +21,18 @@ function initQuickLinksForm() {
         });
     });
 }
+
+function showFilterField() {
+    selector = $('#entity-selector');
```

Comment:
Where is this variable defined? I'm getting js errors in my editor. Variables need to be declared as either var (general) or let (local) in javascript :)
![image](https://user-images.githubusercontent.com/3263440/206076938-c8c0d055-5f3f-45d4-bec7-53d385cfb53f.png)